infrastructure/quick-deploy/aws/variables.tf

```diff
@@ -339,8 +339,8 @@ variable "mongodb" {
     }))
 
     security_context = optional(object({
-      run_as_user = optional(number, 999)
-      fs_group    = optional(number, 999)
+      run_as_user = optional(number, 1001)
+      fs_group    = optional(number, 1001)
     }), {})
```

Comment:
Should we keep this variable ? As 1001 is the default value in Bitnami's Helm chart.

Reply:
I think we should keep the previous value.

Reply:
Currently, with pod and container security context being explicitly defined in the Helm chart, this value seems to break the deployment with a CrashLoopBackOff error for the data pods...